Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -441,6 +441,15 @@ def bottle_formula(f, args:)
         # Ignore matches to source code, which is not required at run time.
         # These matches may be caused by debugging symbols.
         ignores = [%r{/include/|\.(c|cc|cpp|h|hpp)$}]
+
+        # GCC installation can be moved so long as the whole directory tree is moved together:
+        # https://gcc-help.gcc.gnu.narkive.com/GnwuCA7l/moving-gcc-from-the-installation-path-is-it-allowed.
```

Comment:
Unfortunately this advice is only true on Linux, so this exception needs to be Linux-only.